test/test_stats.py

```diff
@@ -63,4 +63,5 @@ def test_response_management():
 
     # Test no responses for path
     k, e = aStats.get_responses("no_path")
-    assert e == False
\ No newline at end of file
+    assert e == False
+    
```

Comment:
For some reason GH is still complaining about a missing newline here... Maybe take one more look and if you can't figure it out, ignore?